src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -1,166 +1,217 @@
-import * as React from "react"
-import { BoxProps, HorizontalOverflow, Pill } from "@artsy/palette"
+import { HorizontalOverflow, Pill } from "@artsy/palette"
 import { NavigationTabs_searchableConnection$data } from "__generated__/NavigationTabs_searchableConnection.graphql"
 import { useAnalyticsContext } from "System/Analytics/AnalyticsContext"
 import { createFragmentContainer, graphql } from "react-relay"
-import { RouterLink, RouterLinkProps } from "System/Router/RouterLink"
+import { RouterLink } from "System/Router/RouterLink"
 import { useIsRouteActive } from "System/Router/useRouter"
 import {
   ActionType,
   ClickedNavigationTab,
   ContextModule,
-  PageOwnerType,
 } from "@artsy/cohesion"
-import { useRouter } from "System/Router/useRouter"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { useTracking } from "react-tracking"
+import { FC, useMemo } from "react"
+import { compact } from "lodash"
 
 export interface NavigationTabsProps {
   searchableConnection: NavigationTabs_searchableConnection$data
   term: string
+  /** TODO: Leave comment explaining why we accept an explicit count for this instead of using the count from the aggregation */
```

Comment:
🤷 Mystery

Reply:
The artwork count/query is a different query than the `searchConnection` that powers the other tabs, so it likely is queried for and passed in specifically.

https://github.com/artsy/force/blob/25e5863775d92154ce9ddc484875390801055b56/src/Apps/Search/SearchApp.tsx#L153 and https://github.com/artsy/force/blob/25e5863775d92154ce9ddc484875390801055b56/src/Apps/Search/Routes/SearchResultsArtworks.tsx#L65 (as compared to the other entities)

Reply:
Why not insert then in mp?